src/css/var/isHiddenWithinTree.js

```diff
@@ -1,9 +1,10 @@
 define( [
 	"../../core",
-	"../../selector"
+	"../../selector",
+	"../../var/isAttached"
```

Comment:
This is incorrect as this is the third parameter but it's used as a second one. You need to put that before `../../selector`. Could you run tests in AMD mode at least in one browser locally? It would catch those issues.

Unfortunately you can't do that via our Node.js setup but most non-AJAX tests should work without a PHP server. Or you can run one via `php -S localhost:7000` from your jQuery repository directory and open `http://localhost:7000`, choosing a module you modified (other than ajax) and enabling the AMD checkbox.

Reply:
Will check this.